### ISSUE TITLE
Document CMultidimensionalScaling

### ISSUE THREAD

**Issue by karlnapf** [shogun-toolbox/shogun#2673]
and other dimensionality reduction methods from Tapkee....

This means: 
 * Link to Wikipedia page
 * Basic math
 * Objective function
 * Computational costs

All this should go to the doxygen class documentation

**Comment by iglesias** [comment]
Part of the content at (tapkee.lisitsyn.me)[http://tapkee.lisitsyn.me/], under Dimension reduction techniques (upper left corner) can be probably recycled for this.

**Comment by curiousguy13** [comment]
Hi, I'm new here and I would like to work on this. 
Also the pull requests have to be for the changes in one file at a time , right?

**Comment by iglesias** [comment]
Nope. You can submit a pr with changes in several files. It is good though to keep prs small and focussing on one task.

**Comment by curiousguy13** [comment]
Oh, okay. So, for this I need to change the documentation in the source files right? for example, for Isomap, I would need to add the changes in src/shogun/converter/Isomap.h ?

**Comment by iglesias** [comment]
Exactly.
